src/player.cpp

```diff
@@ -8977,7 +8977,8 @@ item player::reduce_charges( int position, long quantity )
         debugmsg( "invalid item position %d for reduce_charges", position );
         return ret_null;
     }
-    if( it.reduce_charges( quantity ) ) {
+    it.mod_charges( -quantity );
```

Comment:
No, but `mod_charges()` will show a `debugmsg` in that case (just like `item::reduce_charges()` did).